metadata-ingestion/docs/sources/tableau/tableau_recipe.yml

```diff
@@ -5,7 +5,7 @@ source:
     connect_uri: https://prod-ca-a.online.tableau.com
     site: acryl
     platform_instance: acryl_instance
-    projects: ["default", "Project 2"]
+    project_pattern: ["^default$", "^Project 2$", "^/Project A/Nested Project B$"]
```

Comment:
Can we actually keep this example simple?

project_patterns: ["default", "Project 1", "Project 1/Nested Project 2"]